app/views/locations/show.html.erb

```diff
@@ -42,6 +42,6 @@
   </div>
 </div>
 
-<% if LocationPolicy.new(current_user, @location).deactivate? %>
+<% if @location.scheme.owning_organisation.active? && LocationPolicy.new(current_user, @location).deactivate? %>
```

Comment:
Should this be inside the policy?

Reply:
My thinking here was that I'd usually expect a policy to be restricted to authorisation logic and this feels like it's moving over into validation instead. On the other hand, we already have a very similar rule in our logs policies, so my assumptions about how things work are probably just off-base!